productivity-suite/app/legacy-app/src/components/pages/Login.css

```diff
@@ -0,0 +1,4 @@
+.login-page {
```

Comment:
I made some small tweaks so that we can have a nicer seams outline for the login fragment

before:
![Screenshot 2022-11-08 at 18 39 42](https://user-images.githubusercontent.com/61631103/200649125-f27dbf32-9bdd-4934-b7ae-daffe1184893.png)

after:
![Screenshot 2022-11-08 at 18 39 15](https://user-images.githubusercontent.com/61631103/200649155-3fab2b50-4b47-433c-83a5-0f0dc5b125f1.png)